src/router/index.js

```diff
@@ -0,0 +1,15 @@
+import Vue from "vue";
+import Router from "vue-router";
+import Hello from "../components/Hello";
+
+Vue.use(Router);
+
+export default new Router({
```

Comment:
There's that "default" word again - what is it's job?

But this object makes the routes available to the rest of the app